config.toml

```diff
@@ -53,6 +53,7 @@ anchor = "smart"
 
 [languages]
 [languages.en]
+[languages.en.params]
 title = "Shipwright"
 description = "A framework for building container images on Kubernetes"
 languageName ="English"
```

Comment:
Thanks for the links. I saw the deprecation message, but I am not so sure about this change. I went through the documentation they have, but I couldn't find any complete schema or something like that. Everything I did find kind of pointed into the direction that `title`, `weight`, or `languageName` is under the language section, and **not** under `params`: https://gohugo.io/methods/site/language/. The documentation only really gives `color` as an example for the `params` section and in the internet you find some references that people put `subTitle` under params. Can we double check on that?

Could it be that it would be maybe be more like this:
```suggestion
[languages]
[languages.en]
title = "Shipwright"
languageName ="English"
# Weight used for sorting.
weight = 1

[languages.en.params]
description = "A framework for building container images on Kubernetes"
```

Reply:
@HeavyWombat thanks for catching that! The placement wasn't correct as only custom parameters are to be specified in the languages.xx.params. I have now pushed the modified changes, PTAL.